e2e/docker/test.py

```diff
@@ -0,0 +1,110 @@
+"""Tests for Flower Docker containers."""
+
+import re
+import time
+from typing import Callable, Union
+
+import pytest
+from testcontainers.compose import ContainerIsNotRunning, DockerCompose
```

Comment:
Where is this dependency installed? I could not find it in the `pyproject.toml`.

Reply:
Good point 😅! I've added it now to `pyproject.toml`.